src/components/ArticleListView.jsx

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+import ArticlePreview from 'components/ArticlePreview';
+
+export default class ArticleListView extends React.Component {
+  getInitialState() {
```

Comment:
This should be a pure implementation, meaning it just renders from props. If we don't need to use states at all, it's better.